e2e/cypress/integration/messaging/markdown_preview_inline_image_spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login, set the Show Markdown Preview preference, then go to /
+        cy.apiLogin('user-1');
+        cy.apiSaveShowMarkdownPreviewPreference();
+        cy.visit('/');
+    });
+
+    it('M18714-Markdown preview: inline image', () => {
+        const message = '![make it so](https://i.stack.imgur.com/MNeE7.jpg)';
+
+        cy.visit('/ad-1/channels/town-square');
+
+        // # Get the height before starting to write
+        cy.get('#post_textbox').should('be.visible').clear().then(() => {
+            cy.get('#post-create').then((postArea) => {
+                cy.wrap(parseInt(postArea[0].clientHeight, 10)).as('initialHeight');
+            });
+        });
+
+        // # Post first line to use
+        cy.get('#post_textbox').type(message).type('{shift}{enter}').type(message);
+
+        // # Click on Preview button
+        cy.get('#previewLink').click({force: true});
+
+        // * Image does not overlap Post List
+        cy.get('#post-list').then((postList) => {
+            cy.get('.markdown-inline-img').then((img) => {
```

Comment:
Please wrap this so that we're getting the correct element within `#create_post`
```javascript
cy.get('#create_post').within(() => {
    cy.get('.markdown-inline-img').then((img) => {
        ...
    });
});
```